api/envoy/config/listener/v3/listener.proto

```diff
@@ -247,10 +247,10 @@ message Listener {
   google.protobuf.BoolValue freebind = 11;
 
   // Additional socket options that may not be present in Envoy source code or
-  // precompiled binaries. The socket options can be updated for a listener when
+  // precompiled binaries.
+  // It is not allowed to update the socket options for any existing address if
   // :ref:`enable_reuse_port <envoy_v3_api_field_config.listener.v3.Listener.enable_reuse_port>`
-  // is ``true``. Otherwise, if socket options change during a listener update the update will be rejected
-  // to make it clear that the options were not updated.
+  // is ``false`` to avoid the conflict when creating new sockets for the listener.
```

Comment:
nit: s/avoid the conflict/avoid a mismatch ?

Reply:
I am not a native speaker. But what I mean is to avoid the conflict of try to bind to same address?

Reply:
ahhh got you, thanks for the clarification!  Maybe you can just add that in.. "to avoid attempting to bind to the same address when creating new sockets for the listener".  Thanks!